features/networking/sdn.feature

```diff
@@ -33,18 +33,36 @@ Feature: SDN related networking scenarios
     And the output should contain "Using userspace Proxier"
     """
 
-  # @author bmeng@redhat.com
+  # @author rbrattai@redhat.com
   # @case_id OCP-11286
   @admin
   @destructive
   Scenario: iptables rules will be repaired automatically once it gets destroyed
+    # we do not detect incomplete rule removal since ~4.3, BZ-1810316
+    # so only test on >= 4.3
+    Given the master version >= "4.3"
     Given I select a random node's host
-    And the node iptables config is verified
+    And the node iptables config is checked
+    And the step succeeded
     And the node service is restarted on the host after scenario
+    And I register clean-up steps:
+    """
+    When the node iptables config is checked
+    Then the step succeeded
+    """
+
+    When the node standard iptables rules are removed
+    And 35 seconds have passed
```

Comment:
@rbbratta is it possible to do this check and wait instead of a hard-code wait?

Reply:
We have to wait for default `iptablesSyncPeriod` which is `30` seconds.  Err, well `iptablesSyncPeriod` can be changed post-install so we should query the current value, but it will be `30` seconds 99% of the time.

This step is a negative check, so we have to wait the full period to make sure the rules were not restored.  

Maybe I'll just add a comment explaining the 30 second wait.

```yaml
kubeProxyConfig:
     iptablesSyncPeriod: 30s
     proxyArguments:
          iptables-min-sync-period:
          - 30s
```